src/components/NotificationCard.tsx

```diff
@@ -166,29 +167,29 @@ export const NotificationCard: React.FC<NotificationCardProps> = ({
             </View>
             <View style={{ ...styles.cardRow, flexDirection: 'row' }}>
               {expired ? (
```

Comment:
This `expired` variable is (erroneously?) true if the notification.date is 0. This is happening at the moment, I think because of a bug in the date parsing and modification process when the notification arrives